app/helpers/cafe_helper.rb

```diff
@@ -0,0 +1,42 @@
+module CafeHelper
+  def cafe_worker_form(shift:, current_user: nil, councils: [])
```

Comment:
Ugh. Is this a new ruby feature? :) so now we have to put a colon after all function arguments?